cmp/grid/GridModel.js

```diff
@@ -638,6 +638,56 @@ export class GridModel extends HoistModel {
         }
     }
 
+    /**
+     * Scroll to ensure the provided record or records are visible.
+     *
+     * If multiple records are specified, scroll to the first record and then the last. This will do
+     * the minimum scrolling necessary to display the start of the provided record and as much as
+     * possible of the rest.
+     *
+     * Any provided records that are hidden because their parent rows are collapsed will first
+     * be revealed by expanding their parent rows.
+     *
+     * @param {(StoreRecordOrId|StoreRecordOrId[])} records - one or more record(s) / ID(s) for
+     * which to ensure visibility.
+     */
+    async ensureRecordsVisibleAsync(...records) {
```

Comment:
clever use of ...., but don't think specifying it this way works when users passes in an array as a single argument -- can you flatten the array on line 662?

Reply:
Lodash method castArray() used to process input records.